changelogs/fragments/v10.0.0_bump_ansible_core.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
```

Comment:
You missed the release_summary section.

```
release_summary: Starting from this release, the minimum `ansible-core` version this collection requires is `2.15.0`.
```

Reply:
Well, now that I think of it, we can also add: "The last known version compatible with ansible-core<2.15 is `v9.0.0`.

Reply:
yes yes i am updating that, i was merging the pre-commit PR and then renaming this to Major release PR thats why kept the changes on hold.